src/workerd/io/limit-enforcer.h

```diff
@@ -25,7 +25,7 @@ class IsolateLimitEnforcer {
   // Further customize the isolate immediately after startup.
   virtual void customizeIsolate(v8::Isolate* isolate) = 0;
 
-  virtual ActorCacheSharedLruOptions getActorCacheLruOptions() = 0;
+  virtual const ActorCacheSharedLruOptions getActorCacheLruOptions() = 0;
```

Comment:
This doesn't make sense because it restricts what users of the return value can do with the struct.  For example, users cannot `kj::mv` them.

Reply:
Don't think they're needed here either: https://github.com/cloudflare/workerd/pull/1914/commits/f8b58b5fa86d655035ca4b271362b9bcdb82c149